### PR TITLE
STOR-1988: add enhancement for GCP Filestore Operator WIF

### PR DESCRIPTION
**Pull request by RomanBednar**
cc @openshift/storage

### CODE REVIEW

File: enhancements/storage/gcp-filestore-wif.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+### API Extensions
+
+- No modifications to existing storage resources.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+- Ensure WIF functionality is compatible with Hypershift deployments.
```
Comment (gnufied) [inline enhancements/storage/gcp-filestore-wif.md:143]: Do we support hypershift with gcp file store or gcp-pd deployments?

Comment (RomanBednar) [reply enhancements/storage/gcp-filestore-wif.md:143]: GCP is not yet supported with hosted control plane - removing.

**Comment by RomanBednar** [comment]
I've updated the enhancement with all the currently available information - ready for review/merge. Details about specific test scenarios live under "test plan" document linked in Jira.

**Comment by dobsonj** [comment]
The feature code already merged, I'd rather not let this enhancement rot.
/lgtm

**Pull request merged**